azure-pipelines-release.yml

```diff
@@ -60,9 +60,9 @@ stages:
       vmImage: 'ubuntu-latest'
     steps:
     - task: UsePythonVersion@0
-      displayName: 'Use Python 3.7'
```

Comment:
The upcoming release still needs/is based on 3.7

Reply:
> The upcoming release still needs/is based on 3.7

I don't know what that means in terms of this PR and I know nothing about Azure -- should we revert the changes made to azure-pipelines-release.yml or azure-pipelines-translation.yml or both?

Reply:
Actually, it is needed and fine 👍 

Thanks!